src/formatting.tsx

```diff
@@ -279,6 +281,28 @@ function _varToLabel (str: string) {
 
 export const varToLabel: (str: string) => string = memoize(_varToLabel);
 
+export function getInitials (value?: string | null) {
+  if (!hasStringContent(value)) { return ''; }
+
+  const MAX_CHARS = 3
+    , prefix = value.split(',')[0] || ''
+    , formatted = startCase(prefix)
+    , isValueAllCaps = formatted === upperCase(formatted)
+    , wordArray = formatted.match(RE_WORDS) || [];
+
+  return wordArray
+    .map((word: string, _index: number, all: string[]) => {
+      const notOnlyWord = !!all.length
```

Comment:
Morning re-read: I think this needs to be `length > 1` and a test case added, or this branch / idea removed entirely.

Reply:
Removed this entirely. It wasn't relevant to any test case, which were all generated from real-world data.